src/App.js

```diff
@@ -1,15 +1,54 @@
 import React, { Component } from 'react';
 import './App.css';
-import Amount from './Amount';
 
 class App extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      amount: 0,
+    };
+  }
+
+  onIncrement = () => {
+    this.setState(state => ({ amount: state.amount + 1 }));
+  };
+
+  onDecrement = () => {
+    this.setState(state => ({ amount: state.amount - 1 }));
+  };
```

Comment:
There's a lot more code in App now.  It would be great if we could push this down to the Amount and Currency components somehow.